### ISSUE TITLE
{"message":"The provided secret has expired."}

### ISSUE THREAD

**Issue by guillaume-hb** [chapter-three/next-drupal#493]
When we got on View on a drupal content to get the preview, sometimes it works but take a long time, and few times after, I've got this error :

{"message":"The provided secret has expired."}

It seems that the token is not refreshed well from the nextjs side

**Comment by shadcn** [comment]
Yes. Right now, a new token is created when you visit the View page. It does not refresh automatically. You have to refresh the page to get a new token.

You could work around this by setting a larger expiration time for preview secrets.

**Comment by guillaume-hb** [comment]
Thanks for your answer, but on a AWS EC2 instance, it makes it crash, so the token could not be refresh.
It looks like it takes too much time so get a response, so it crashes

Any idea to fix this bug ?

Thanks

**Comment by balintbrews** [comment]
I think this issue is a duplicate of #422.
